src/molecules/Select/ComboBox/ComboBox.tsx

```diff
@@ -14,5 +14,11 @@ export type ComboBoxProps<T extends SelectOptionRequired> = CommonSelectProps &
 export function ComboBox<T extends SelectOptionRequired>(
   props: ComboBoxProps<T>
 ) {
+  if (props.groups && props.onAddItem) {
+    throw new Error(
+      "Using 'onAddItem' is only supported in lists and not groups"
```

Comment:
Maybe this should include that it is the ComboBox component throwing the error. 
Unless that maybe comes through in the full error message we would see? 🤔

Reply:
Updated